Examples/Instances/Instance-example.fsh

```diff
@@ -1,4 +1,27 @@
 // @Name: Instance Example
+// @Description: An example of an instance of a Patient resource
 
-// Nothing yet! 
-// Help wanted -- your example can go here. 
\ No newline at end of file
+Instance: PatientExample
+InstanceOf: Patient
+Description: "Example of Patient"
+* name.family = "Anyperson"
+* name.given[0] = "John"
+* name.given[1] = "B."
```

Comment:
Have we intentionally decided to always use explicit indices in examples?  Or is this something we should discuss?

Reply:
Switching to soft indices